chain/consensus/mir/manager.go

```diff
@@ -367,31 +361,31 @@ func (m *Manager) initCheckpoint(params trantor.Params, height abi.ChainEpoch) (
 
 // GetSignedMessages extracts Filecoin signed messages from a Mir batch.
 func (m *Manager) GetSignedMessages(mirMsgs []Message) (msgs []*types.SignedMessage) {
```

Comment:
Do we still need this function? It seems to have been replaced by the state manager's `getSignedMessages`.
If this one is not necessary, let's remove it.